scripts/validate_container.sh

```diff
@@ -48,12 +48,12 @@ available_backends=`\
     do
         libEM=$(cat $file | grep "LIBRARY_MODE_EXECUTION_MANAGER=")
         if grep -q "LIBRARY_MODE_EXECUTION_MANAGER=" $file ; then 
-          continue
+        continue
```

Comment:
Suggested change to restore spacing back to what it was before the interim changes.
```suggestion
          continue
```